docs/javascript-action.md

```diff
@@ -6,57 +6,55 @@ The [javascript-template](https://github.com/actions/javascript-template) repo c
 
 Navigate to https://github.com/actions/javascript-template
 
-Click on `Use this template` to create the repo for your action.
+Click on `Use this template` to create the repo for your action.  Provide a name such as `myaction`.
 
 ![template](assets/node12-template.png)
 
-Complete creating your repo and clone the repo.
+# Clone and Update
 
-> NOTE: The location of the repo will be how users will reference your action in their workflow file with the using keyword.
-
-e.g. To use https://github.com/actions/setup-node, user's will author:
-
-```yaml
-steps:
-    uses: actions/setup-node@master
+```bash
+$ git clone <repolocation>
+$ cd myaction
 ```
 
+Update the `author` element in the package.json file.
+
 # Dev Workflow
 
 The workflow below describes one possible workflow with a branching strategy.  Others exist.  
 
-> Key Point: the branch that users reference in their workflow files should reference an action that has **only** the production dependencies. 
+> Key Point: the branch that users reference in their workflow files should reference an action from a distribution branch that has **only** the production dependencies. 
```

Comment:
This reads a bit confusing, can we add an example? "For instance, you would want to reference X and not Y".

Reply:
This is covered further down in the doc in the section for creating a distribution branch.

Reply:
The following paragraph also discussed a concrete distribution branch of releases/v1.  It also clarifies that the runner will download and run from that distribution branch.